scripts/import_dvv.sql

```diff
@@ -0,0 +1,38 @@
+-- Insert buildings to jkr_rakennus
+insert into jkr.rakennus (prt, kiinteistotunnus, onko_viemari, geom, kayttoonotto_pvm, rakennuksenkayttotarkoitus_koodi, rakennuksenolotila_koodi)
+select 
+    rakennustunnus as prt,
+    "sijaintikiinteistön tunnus" as kiinteistotunnus,
+    viemäri::boolean as onko_viemari,
+    ST_GeomFromText('POINT('||itä_koordinaatti||' '||pohjois_koordinaatti||')', 3067) as geom,
+    to_date("käytössä_olotilanteen muutospäivä"::text, 'YYYYMMDD') as kayttoonotto_pvm,
+    "käyttö_tarkoitus" as rakennuksenkayttotarkoitus_koodi,
+    "käytös_säolo_tilanne" as rakennuksenolotila_koodi
+from jkr_dvv.rakennus
+on conflict do nothing
+;
+
+-- Insert streets to jkr_osoite.katu
+-- jkr_osoite.kunta must be filled in the database by running import_posti.sql first!
+insert into jkr_osoite.katu (katunimi_fi, katunimi_sv, kunta_koodi)
```

Comment:
In performance wise this query could be optimized slightly. Now this tries to do over 90 000 inserts and most of the time the street is already in the database.
We could prefilter only unique streets with:
```sql
select distinct
    "kadunnimi suomeksi" as katunimi_fi,
    "kadunnimi ruotsiksi" as katunimi_sv,
    sijainti_kunta as kunta_koodi
from jkr_dvv.osoite
```

Reply:
Certainly the queries could be improved, but if the query is run once a year and it takes a few seconds, how much is it worth to spend time optimizing such a query?